LayoutTests/resources/testharnessreport.js

```diff
@@ -14,8 +14,7 @@
 if (self.testRunner) {
     testRunner.dumpAsText();
     testRunner.waitUntilDone();
-    testRunner.grantWebNotificationPermission("http://localhost:8800");
-    testRunner.grantWebNotificationPermission("http://web-platform.test:8800");
+    testRunner.grantWebNotificationPermission(self.origin);
```

Comment:
Would be nice to describe in the commit message what you're doing here, or split this change out.

Reply:
I can't split it out. Some of the notification tests run over https now so we need to grant access to the https host too. Allowing access to self.origin is the most relying way to grant access to the current origin, no matter what origin the test runs on.

Reply:
I'll update the change log to explain.